policy/README.md

```diff
@@ -0,0 +1,353 @@
+    "description": "LADOT Idle Time Limitations",
+    "start_date": 1552678594428,
+    "end_date": null,
+    "prev_policies": null,
+    "rules": [{
+        "name": "Greater LA (rentable)",
+        "rule_type": "time",
+        "rule_units": "hours",
+        "geographies": ["b4bcc213-4888-48ce-a33d-4dd6c3384bda"],
+        "statuses": ["available", "reserved"],
+        "vehicle_types": ["bicycle", "scooter"],
+        "maximum": 72
+    }, {
+        "name": "Greater LA (non-rentable)",
+        "rule_type": "time",
+        "rule_units": "hours",
+        "geographies": ["12b3fcf5-22af-4b0d-a169-ac7ac903d3b9"],
+        "statuses": ["unavailable", "trip"],
+        "vehicle_types": ["bicycle", "scooter"],
+        "limit": 24
+    }]
+}
+```
+
+Speed limits example.  Fifteen MPH in greater LA, 10 MPH on Venice Beach on Saturday/Sunday from noon til midnight.
+
+```
+{
+    "policy_id": "95645117-fd85-463e-a2c9-fc95ea47463e",
+    "name": "Speed Limits",
+    "description": "LADOT Pilot Speed Limit Limitations",
+    "start_date": 1552678594428,
+    "end_date": null,
+    "supersedes": null,
+    "rules": [{
+        "name": "Greater LA",
+        "rule_type": "speed",
+        "rule_units": "mph"
+        "geographies": ["b4bcc213-4888-48ce-a33d-4dd6c3384bda"],
+        "states": ["trip"],
+        "vehicle_types": ["bicycle", "scooter"],
+        "maximum": 15
+    }, {
+        "name": "Venice Beach on weekend afternoons",
+        "rule_type": "speed",
+        "rule_units": "mph",
+        "geographies": ["ec551174-f324-4251-bfed-28d9f3f473fc"],
+        "states": ["trip"],
+        "vehicle_types": ["bicycle", "scooter"],
+        "days": ["sat", "sun"],
+        "start_time": "12:00",
+        "end_time": "23:59",
+        "maximum": 10,
+        "messages": {
+            "en-US": "Remember to stay under 10 MPH on Venice Beach on weekends!”,
+            "es-US": "¡Recuerda permanecer menos de 10 millas por hora en Venice Beach los fines de semana!"
+        },
+    }]
+}
+```
+
+Parking example combining place and time.  Maximum 300 vehicles; each can stay up to 48 hours.
+
+```
+{
+    "policy_id": "283dc770-7d59-4880-844a-307681d7542f",
+    "name": "Parking",
+    "description": "West 34th Street Parking",
+    "start_date": 1552678594428,
+    "end_date": null,
+    "prev_policies": null,
+    "rules": [{
+      "name": "West 34th Street Parking Time Limit",
+      "rule_type": "time",
+      "rule_units": "hours", 
+      "geographies": ["f3637b37-ee80-46e2-b01f-3a7f7f819dd9"],
+      "statuses": ["available", "trip"],
+      "vehicle_types": ["bicycle", "scooter"],
+      "maximum": 48
+    }, {
+      "name": "West 34th Street Parking Capacity",
+      "rule_type": "count",
+      "geographies": ["f3637b37-ee80-46e2-b01f-3a7f7f819dd9"],
+      "statuses": ["available", "trip"],
+      "vehicle_types": ["bicycle", "scooter"],
+      "maximum": 300
+    }]
+  }
+}
+```
+
+More examples to come.  :)
```

Comment:
Remove line from spec here. Commentary.

Reply:
Agreed, removing.

Reply:
Amendment, we're going to add all of the current Policy objects in use by LADOT for compliance measurements.